client/src/modules/events/pages/eventPage.tsx

```diff
@@ -87,10 +87,14 @@ export const EventPage: NextPage = () => {
     const confirmOptions = options?.invited
       ? {
           title: 'You have been invited to this event',
-          body: 'Would you like to attend?',
+          body: `
+          Would you like to attend?<br/>
+          Note: if you haven't joined event's chapter, you will become member when RSVPing
```

Comment:
```suggestion
          Note: joining this event will make you a member of the event's chapter.
```
Let's go with this for now.